### PR TITLE
Setup react app

### PR DESCRIPTION
**Pull request by jonas-masharqa**
# PR content 
[PT-link](https://www.pivotaltracker.com/story/show/169262591)
Scaffolds React Application

## This PR Contains
- Scaffolded React application.
- Cleans up files.

### CODE REVIEW

**Review by piavW — APPROVED** [review]

**Review by AlexSaxena — APPROVED** [review]

**Review by clalil — APPROVED** [review]

**Review by miyeaier — APPROVED** [review]

**Review by miyeaier — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
```
Comment (faraznaeem) [inline src/index.js]: Please clean out the comments

File: src/serviceWorker.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+// register() is not called by default.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on subsequent visits to a page, after all the
+// existing tabs open on the page have been closed, since previously cached
+// resources are updated in the background.
+
+// To learn more about the benefits of this model and instructions on how to
+// opt-in, read https://bit.ly/CRA-PWA
```
Comment (faraznaeem) [inline src/serviceWorker.js]: Please clean out the comments

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (clalil) [reply src/index.js]: Changed accordingly.

File: src/serviceWorker.js (hunk 2 above)
Comment (clalil) [reply src/serviceWorker.js]: Changed accordingly.

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**